### ISSUE TITLE
Feedstock info in anaconda package landing page

### ISSUE THREAD

**Issue by CJ-Wright** [conda-forge/conda-forge.github.io#515]
Would it be possible to put some info on where to find the feedstock repo on the anaconda package landing page? This way if users have issues with the package and go to the anaconda page they can then follow a link back to the feedstock (where they can put up an issue).

**Comment by jakirkham** [comment]
I completely agree @CJ-Wright. Had started some work on this at the `conda-build` level in PR ( https://github.com/conda/conda-build/pull/2489 ), but have lacked any time to address follow-up comments. If you or anyone else is interested, would welcome the help on finishing that work out.

**Comment by pelson** [comment]
Personally, I think we should probably have this information as a document in each of the feedstocks, and that each time the CI runs on master it ensures the "description" field is up-to-date on anaconda.org.

That we, we can template the information using conda-smithy.

I guess fundamentally, my issue with having it in the package description inside the conda binary is that anaconda.org is 1-2-1 for packages <> description, but 1-2-many for packages <> package binaries. Therefore if the package binaries have different descriptions, which one should anaconda.org show.
